jobs/waiting-for-jobs.rst

```diff
@@ -0,0 +1,216 @@
+
+Here are several examples.  In this first one, we submit a simple job that sleeps for 30 seconds then runs ``/bin/true``.  Afterwards, we pass the jobid to ``flux job status`` and wait for it to return when the job has finished.  After it has completed we can see that the exit code from ``flux job status`` is ``0``, as we expect from ``/bin/true``.
+
+.. code-block:: console
+
+    $ flux submit -n1 bash -c "sleep 30; /bin/true"
+    ƒLUebmCK
+    $ flux job status ƒLUebmCK
+    <we wait a little bit waiting for the job to finish>
+    $ echo $?
+    0
+
+If we run the above job with ``/bin/false`` instead of ``/bin/true``.
+
+.. code-block:: console
+
+    $ flux submit -n1 bash -c "sleep 30; /bin/false"
+    ƒeGz9fYs
+    $ flux job status ƒeGz9fYs
+    <we wait a little bit waiting for the job to finish>
+    $ echo $?
+    1
+
+The result is identical to the first example except the exit code from ``flux job status`` is ``1``, which is what we expect from running ``/bin/flase``.
+
+Finally, lets pass both jobids from above to ``flux job status``.
+
+.. code-block:: console
+
+    $ flux job status ƒLUebmCK ƒeGz9fYs
+    $ echo $?
+    1
+
+You'll notice two things about this example.  First, the command returns immediately.  This is because the two jobs have already completed.  Second, the exit code is ``1``, which is the largest exit code of the two jobs passed to ``flux job status`` (one ran ``/bin/true`` and the other ran ``/bin/false``).
+
+-------------
+Flux Job Wait
+-------------
+
+``flux job wait`` behaves similarly to ``flux job status`` but there are some differences in using it, which come with pros and cons.
+
+The most notable difference is that in order to use ``flux job wait``, jobs must be passed the ``waitable`` flag.  Any job that is not passed the ``waitable`` flag will not work with ``flux job wait``.  In addition, the ``waitable`` flag can only be used in user Flux instances (i.e. non-system instances).  User Flux instances are usually started via ``flux alloc`` or ``flux batch``.
+
+Here's a simple example of using ``flux job wait``.  It's very similar to the example from before, where run sleep for 30 seconds then run ``/bin/true``.
+
+.. code-block:: console
+
+    $ flux submit --flags waitable -n1 bash -c "sleep 30; /bin/true"
+    ƒ4btMovw
+    $ flux job wait ƒ4btMovw
+    <we wait a little bit waiting for the job to finish>
+    $ echo $?
+    0
+
+Note that when submitting the job, we submitted it with the ``waitable`` flag via ``--flags waitable``.
+
+This doesn't really show us anything special, it seems to be the same as ``flux job status``.  Lets now look at the major advantages of ``flux job wait``.
+
+Perhaps the biggest advantage of ``flux job wait`` is that apriori knowledge of jobids is not necessary.  If ``flux job wait`` is specfied without any jobids, it will wait for the first job that completes amongst all of the jobs you have submitted via the ``waitable`` flag.
```

Comment:
So this is a flag that plops a job into a group you want to wait for? So it's appropriate for a scoped run of some kind?